crates/ruff_cli/src/commands/run.rs

```diff
@@ -144,6 +147,44 @@ pub fn run(
             acc
         });
 
+    debug!("{:#?}", diagnostics.imports);
+    let mut cycles: FxHashMap<Arc<str>, FxHashSet<Vec<Arc<str>>>> = FxHashMap::default();
+
+    for (path, package, settings) in
+        paths
+            .iter()
+            .filter_map(|entry| entry.as_ref().ok())
+            .map(|entry| {
+                let path = entry.path();
+                let package = path
+                    .parent()
+                    .and_then(|parent| package_roots.get(parent))
+                    .and_then(|package| *package);
+                let settings = resolver.resolve_all(path, pyproject_strategy);
+                (path, package, settings)
+            })
```

Comment:
Nit: I first assumed that this is the body of the `for` loop because of how it is nested. It could make sense to assign the result of this expression to a variable and then use it in the for loop

Reply:
Hm, I had to format it that way due to `cargo fmt` I think, the variables that are to be used in the actual for loop are in the line where the for loop starts?